src/core_plugins/kibana/public/discover/embeddable/search_embeddable.js

```diff
@@ -77,11 +85,23 @@ export class SearchEmbeddable extends Embeddable {
   initializeSearchScope() {
     this.searchScope = this.$rootScope.$new();
 
-    this.pushContainerStateParamsToScope();
-
     this.searchScope.description = this.savedSearch.description;
     this.searchScope.searchSource = this.savedSearch.searchSource;
 
+    const timeRangeSearchSource = this.searchScope.searchSource.new();
+    timeRangeSearchSource.filter(() => {
+      return getTime(this.searchScope.searchSource.index(), this.timeRange);
+    });
+
+    const filtersSearchSource = this.searchScope.searchSource.new();
+    filtersSearchSource.inherits(timeRangeSearchSource);
+    filtersSearchSource.filter([]);
```

Comment:
Is it actually necessary we set those default values, before we push the real values from the container state?